sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/issue/Issue.java

```diff
@@ -69,4 +70,10 @@ interface Flow {
    * @since 9.2
    */
   boolean isQuickFixAvailable();
+
+  /**
+   * This will be filled in if the how to fix section detects a specific context.
+   * @since 9.6
+   */
```

Comment:
```suggestion
  /**
   * The optional rule description section context key, in case the analyzer detects a context for the issue. 
   * The key will match the one present in {@link Context#getKey()}.
   * @since 9.6
   */
```